src/vtk_calculator.py

```diff
@@ -85,26 +88,22 @@ def main():
     calc.AddCoordinateScalarVariable("y", 1)
     calc.AddCoordinateScalarVariable("z", 2)
     if args.diff:
-        calc.SetFunction('{}-({})'.format(intag, args.function))
-        logging.info("Evaluated '{}-({})' on the mesh.".format(intag, args.function))
-    else:
+        # Check VTK file has dataname
+        if not vtk_dataset.GetPointData().HasArray(intag):
+            logging.warning("Given mesh has no data for \"{}\".\n ABORTING! \n".format(intag))
```

Comment:
```suggestion
            logging.warning("Given mesh has no data with the given name \"{}\".\nABORTING!\n".format(intag))
```
Maybe also add the given related mesh name here.